code/game/objects/effects/decals/posters/asters.dm

```diff
@@ -0,0 +1,58 @@
+// Asters Guild faction posters
+/datum/poster/asters/asterite
+	name = "Voyage of The Asterites"
+	desc = {"A flourished bit of symbolism representing the first Asterite's great exodus from Humanity's origin, now posed to to inherit the stars themselves. \
+	At least, that's what the small text says."}
+	icon_state = "asters_asterite"
+
+/datum/poster/asters/blurb
+	name = "The Asterite's Message"
+	desc = "A poster labeling the good will and intentions of The Aster's Guild under American principles of free will and enterprise."
+	icon_state = "asters_blurb"
+
+/datum/poster/asters/social_pyramid
+	name = "The Modern Atlas"
+	desc = "At the pyramid's bottom comes the great employer, who holds up society upon his back. To the top, the worker who reaps its benefits."
+	icon_state = "asters_social_pyramid"
+
+/datum/poster/asters/cev_eris
+	name = "CEV Eris"
+	desc = "A poster celebrating the maiden voyage of the Cosmic Exploration Vessel \"Eris\"."
+	icon_state = "asters_cev_eris"
+
+/datum/poster/asters/rock_and_stone
+	name = "Rock & Stone"
+	desc = "\"Rock and stone!\""
+	icon_state = "asters_rock_and_stone"
+
+/datum/poster/asters/workplace_gaming
+	name = "Workplace Gaming"
+	desc = {"This poster wishes to remind you that in the event this reality is a virtual simulation, you will be scored on workplace output, not your paycheck. \
+	\"So lets go and highscore!\""}
+	icon_state = "asters_workplace_gaming"
+
+/datum/poster/asters/mine_ore_1
+	name = "Mine Ore"
+	desc = {"This poster wishes to remind you that an astral miner's work is what allows industry to function, and if you're among them to take pride within your work - \
+	\"but not too much!\""}
+	icon_state = "asters_mine_ore_1"
+
+/datum/poster/asters/mine_ore_2
+	name = "Mine More Ore"
+	desc = "\"Remember miners, do not die!\""
+	icon_state = "asters_mine_ore_2"
+
+/datum/poster/asters/move_freight_1
+	name = "Move Freight"
+	desc = "This poster explains the important aspect within Humanity's logistical supply chain the humble technician's role is."
```

Comment:
Guild is more about making money than whatever humanity needs, but that might be ok as propaganda, not sure.